packages/ia-components/sandbox/theatres/with-youtube-spotify/utils/flatten-album-data.js

```diff
@@ -56,23 +56,31 @@ const flattenAlbumData = (metadata, playFullIAAudio) => {
     files: allFiles
   } = metadata;
   const { collection, identifier: itemIdentifier } = albumMetadata;
-  const fileDirectoryPrefix = `${server.includes('://') ? '' : 'https://'}${server}${directoryPath}/`
+  const fileDirectoryPrefix = `${server.includes('://') ? '' : 'https://'}${server}${directoryPath}/`;
 
   let countSampleMP3 = 0;
   let countFullMP3 = 0;
   let countOriginalAudioFiles = 0;
+  let countSegmentedTracks = 0;
   /**
    * Take original item's file list
    * & only return the files we are interested in
    */
   const slimFiles = allFiles.reduce((acc = [], file) => {
-    const { name: fileName, source = '' } = file;
+    const { name: fileName, source = '', original = '' } = file;
     const isValidAudio = isValidAudioFile(fileName);
     const isValidImage = isValidImageFile(fileName);
     const isNeededFile = isValidAudio || isValidImage;
     const fileIsDerived = source === 'derivative';
     const isSampleMP3 = fileName.match('_sample.mp3');
     const isMP3 = fileName.match('.mp3');
+    const isVBRMP3 = fileName.match('_vbr.mp3');
```

Comment:
It's not as likely for '_vbr.mp3' and '_sample.mp3', but these match strings may appear in the file name and not necessarily be the end of the file name plus the extension. Instead, you can switch these out for regexes to only match these patterns at the end of the string, like `fileName.match(/_vbr\.mp3$/)` to ensure you're looking at the end of the file name and extension.

Reply:
Now that I'm further into the review, it looks like you've already got a utility function for this exact matcher as `isOlderDerivedMP3`

Reply:
totally, will update to use regex 👍